File: 240522/소수 판별기/prime-number-discriminator.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        char result = 'P';

        for(int i = 2; i <= Math.sqrt(n); i++){
            if(n % i == 0){
                result = 'C';
                break;
            }
        }

        System.out.println(result);
    }
}

File: 240522/소수 판별기/prime-number-discriminator.js
const fs = require('fs');
const n = Number(fs.readFileSync(0).toString().trim());

function isDemical (number){
    for(let i = 2; i <= Math.sqrt(number); i++){
        if(number % i === 0){
            return false;
        }
    }
    return true;
}

const result = isDemical(n) ? 'P' : 'C';

console.log(result);